.changeset/honest-turkeys-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/gm-token-adapter': patch
```

Comment:
Can we make this a minor, as it does technically change the functionality?

Reply:
Actually noticed that `WBTC.b` isn't uppercase, so this doesn't make sense. Closing!